src/com/sap/piper/DefaultValueCache.groovy

```diff
@@ -17,12 +17,26 @@ class DefaultValueCache implements Serializable {
         }
     }
 
-    static getInstance(){
-        return instance
+    static DefaultValueCache getInstance(Binding scriptBinding){
+        if (instance) {
+            return instance
+        }
+        if(scriptBinding?.hasVariable("defaultValueCacheInstance")) {
+            instance = scriptBinding?.getProperty("defaultValueCacheInstance")
+            return instance
+        }
```

Comment:
Return is missing in case both if are false

Reply:
Better return null in that case